app/components/buy-now.ts

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
```

Comment:
Looks like I'm not good at `gts` 😅  I switch to ts/hbs mainly because I was not able to use `{{on }}` modifier in the `<template>`part of the gts file.

Reply:
I got you! I'll PR off of this to get gts working for this